src/EventListener/DataContainer/ShortlinkLabelListener.php

```diff
@@ -29,7 +29,7 @@ public function __invoke(array $row, string $label, DataContainer $dc, array $co
     {
         $url = $this->generator->generate((int) $row['id'], $row['alias'] ?? null);
 
-        $columns[0] = sprintf('<a href="//%s" target="_blank">%s</a>', $url, $url);
+        $columns[0] = sprintf('<a href="%s" target="_blank">%s</a>', $url, preg_replace('/^https?:\/\//', '', $url));
```

Comment:
why the preg_replace?

Reply:
why would you remove that from a link?

Reply:
aaah… thats only the **display link** in the backend 👍

Reply:
Exactly that.